### ISSUE TITLE
Documentation request: how to build and run tests

### ISSUE THREAD

**Issue by AdelKS** [felixguendling/cista#171]
Hello!

I tried to add the project to my IDE (Qt Creator) and it kept telling me there's nothing to compile :thinking: 

Then I tried the simple CLI approach
```
git clone https://github.com/felixguendling/cista.git
cd cista
mkdir build
cd build
cmake ..
cmake --build .
```
And nothing happens, am I missing something ? I have near to zero knowledge in `cmake` (went for `meson` directly in my own projects)

**Comment by felixguendling** [comment]
I think the reason is `EXCLUDE_FROM_ALL` in CMakeLists.txt.
You need to give it a specific target to build. For example: `cmake --build . --target cista-test`

**Comment by AdelKS** [comment]
Ah yeah that works thanks! is it possible to support `--target all` ? It seem that my IDE is trying to use that instead, is it common ?

**Comment by felixguendling** [comment]
You mean to remove the `EXCLUDE_FROM_ALL` from the executable targets? That would be `cista-test` and `cista-single-header-test`? Or also the amalgamation tool? I think it's best if the user selects what they would like to run. Otherwise, you end up building a lot of stuff again and again that you don't use (e.g. `cista-test` and `cista-single-header-test` are redundant, the latter one is only required to check that the release single header works).

**Comment by AdelKS** [comment]
I am not sure what I am asking, but I suppose to have this to work

```
git clone https://github.com/felixguendling/cista.git
cd cista
mkdir build
cd build
cmake ..
cmake --build . --target all
```

This what I saw my IDE doing. Is it possible to define the target "all" so the above works ? or is "all" a `cmake` specific target. In any case, I figured out how to build only `cista-test` so we are all good ! 

We can close this issue without doing anything. Although I suppose a small addition the the readme with 

```
git clone https://github.com/felixguendling/cista.git
cd cista
mkdir build
cd build
cmake ..
cmake --build . --target cista-test
```

is welcome
